Packs/ClarotyXDome/IncidentFields/incidentfield-Claroty_xDome_Device_First_Seen.json

```diff
@@ -0,0 +1,32 @@
+{
+    "id": "incident_clarotyxdomedevicefirstseen",
+    "version": -1,
+    "modified": "2024-04-18T10:35:36.061254113Z",
+    "name": "Claroty xDome Device First Seen",
+    "ownerOnly": false,
+    "description": "The date and time a device's NIC was first seen",
+    "cliName": "clarotyxdomedevicefirstseen",
+    "type": "multiSelect",
```

Comment:
is this should be a multi-select?

Reply:
This depends on your values from the fetched JSON of the alert. If you are mapping multiple values use the multi-select option then :) just wanted to make sure that this is intentional.